src/custom/state/price/updater.ts

```diff
@@ -73,10 +58,14 @@ function quoteUsingSameParameters(currentParams: FeeQuoteParams, quoteInfo: Quot
 /**
  *  Decides if we need to refetch the fee information given the current parameters (selected by the user), and the current feeInfo (in the state)
  */
-function isRefetchQuoteRequired(currentParams: FeeQuoteParams, quoteInformation?: QuoteInformationObject): boolean {
+function isRefetchQuoteRequired(
+  isLoading: boolean,
+  currentParams: FeeQuoteParams,
+  quoteInformation?: QuoteInformationObject
+): boolean {
   // If there's no quote/fee information, we always re-fetch
   // we need to check that there is also no error otherwise this will loop
-  if (!quoteInformation || (!quoteInformation.fee && !quoteInformation.error)) {
```

Comment:
we can simplify this. Since `getNewQuoteStart` already initializes the `quoteInformation` this won't be an issue anymore